common/geth/handle_error.go

```diff
@@ -65,15 +66,15 @@ func (f *FailoverController) handleError(err error) (NextEndpoint, ImmediateActi
 		var rpcError rpc.Error
 		if errors.As(err, &rpcError) {
 			ec := rpcError.ErrorCode()
-			f.Logger.Info("[JSON RPC Response Error]", "Error Code", ec, "Error", rpcError)
-			// we always attribute JSON RPC error as sender's fault, i.e no connection rotation
-			return CurrentRPC, Return
+			f.Logger.Warn("[JSON RPC Response Error]", "numRPCFfault", f.numberRpcFault, "errorCode", ec, "err", rpcError)
```

Comment:
Is it possible to extract top level domain (ie strip api token) from url and include it in message so that oncall knows exactly which provider failed?

Reply:
addressed, please take a look